Lib/test/test_file.py

```diff
@@ -154,6 +154,20 @@ def testModeStrings(self):
                 f.close()
                 self.fail('%r is an invalid file mode' % mode)
 
+    def testStdin(self):
+        if sys.platform == 'osf1V5':
+            # This causes the interpreter to exit on OSF1 v5.1.
+            self.skipTest(
+                ' sys.stdin.seek(-1) may crash the interpreter on OSF1.'
+                ' Test manually.')
+
+        if not sys.stdin.isatty():
+            # Issue 14853: stdin becomes seekable when redirected to a file
+            self.skipTest('stdin must be a TTY in this test')
+
+        self.assertRaises((IOError, ValueError), sys.stdin.seek, -1)
```

Comment:
I prefer to see the two line with self.assertRaises(...): syntax so that the calls being made look like actual calls rather than come from parameters.